### ISSUE TITLE
[Bug]: Highlight the import files

### ISSUE THREAD

**Issue by Lzhiyong** [bobbylight/RSyntaxTextArea#461]
**Description**
the `CPlusPlusTokenMaker.flex` and `CTokenMaker.flex` can't parse header files correctly.

like this:  <xxx.h> 

![IMG_20220929_190938](https://user-images.githubusercontent.com/36750502/193019615-a057618b-88e6-46ec-8c12-6e7eef06abe0.jpg)

**Expected behavior**
Can correctly highlight header files.

like this:

![IMG_20220929_191008](https://user-images.githubusercontent.com/36750502/193020437-4de8a786-5874-4c7d-97a2-78a47fe5e661.jpg)

**Java version**
openjdk-17

**Additional context**
I can't fix this as I'm not familiar with the `Jflex` syntax.
see the [cpp.flex](https://github.com/luoyesiqiu/Lexer/blob/master/cpp.flex)

Pseudo code:
```flex
/* Add state to import files */
%state IMPORT_FILE

%%

<YYINITIAL> {
    ...
    /* Preprocessor directives */
    "#"{WhiteSpace}*{PreprocessorWord}    { start = zzMarkedPos-2; yybegin(IMPORT_FILE); }
  
    <IMPORT_FILE>{
        /* <xxx.h> or <xxx> as the StringLiteral  */
	{WhiteSpace}\*<{AnyChrChr}>    { int temp=zzStartRead; addToken(start,zzStartRead-1, Token.PREPROCESSOR); addToken(temp,zzMarkedPos-1, Token.LITERAL_CHAR); start = zzMarkedPos; }
	\n			{ addToken(start,zzStartRead-1, Token.LITERAL_CHAR); return firstToken; }
	<<EOF>>			{ addToken(start,zzStartRead-1, Token.LITERAL_CHAR); return firstToken; }
    }

}
```

**Comment by Lzhiyong** [comment]
@bobbylight  thanks

**Comment by Lzhiyong** [comment]
![IMG_20221026_172140](https://user-images.githubusercontent.com/36750502/197989005-7f6ab7fc-5644-4026-94a5-bd41eb1a54f9.jpg)

@bobbylight, there are still some problems.
```h
// contains /
#include <sys/types.h>

// contains multiple spaces between # and include
# include <stdio.h>
#        include <math.h>

```
I think the correct expression should be `"#"[ \t\f]*"include <"[A-Za-z0-9_./]+">"`

Is there a way to highlight the import file? like `Java`, `kotlin`...
```java
// highlight the java.util.ArrayList
import java.util.ArrayList;
```

**Comment by bobbylight** [comment]
Thanks, I've addressed the whitespace and `/` and `-` characters and published a 3.3.1-SNAPSHOT build to Sonatype with these changes.

As for your second question, are you asking for a way to highlight the imported file a different color than the surrounding `<` and `>` characters? Or are you asking if we can be smarter and highlight files known to be in the standard library?

**Comment by Lzhiyong** [comment]
> Thanks, I've addressed the whitespace and `/` and `-` characters and published a 3.3.1-SNAPSHOT build to Sonatype with these changes.
> 
> As for your second question, are you asking for a way to highlight the imported file a different color than the surrounding `<` and `>` characters? Or are you asking if we can be smarter and highlight files known to be in the standard library?

@bobbylight thanks, Specify package name  color
for example:
```java
// like include 

// highlight the 'com.xxx.package_name'
package com.xxx.package_name;

// highlight the 'com.xxx.classs_name'
import com.xxx.pkgname.classs_name;

```
